### ISSUE TITLE
Add non image version of tiles

### ISSUE THREAD

**Issue by KeeganW** [KeeganW/ti4#108]

**Comment by adrianloy** [comment]
Does this refer to a visualization similiar to https://joepinion.github.io ? I would love such a feature, its right now a big reason for me to work on both environments in parallel

**Comment by KeeganW** [comment]
Correct! I am still working out a way to make the visualizations unique, but communicate the right information for map designers.
